### ISSUE TITLE
A proposed revisit on the structure of msociety and its communication channels

### ISSUE THREAD

**Issue by muhdmirzamz** [SGTinkers/msociety-blueprint#1]
I would like to propose a revisit to our channels’ different purposes as I’ve seen quite a few instances where our members do not exactly know where to post. 

These are some questions worth exploring:
1) Is setting up another telegram channel for controversial/political discussions okay?
2) Some of us may still not know the existence of the other channels. 
3) Some topics are in the grey area so, in the name of convenience, there’s potential for cross-posting across channels. 

Personal opinion: 
I’ve lately been exposed to a really good Discord system for a community and I’m starting to view it as a much, much better option. I guess it’s the visibility of the different channels and the strict purpose of the said channels that really allow for more fluid flow of a conversation.

**Comment by ruqqq** [comment]
1. I think we were consciously making an effort not too have many splinter groups to avoid having groups with low activity. For example, the ML group is not very active.

We have also seen in the past that a lot of conversations ended up being cross cutting which was why the main group purpose was changed to be a "lobby". Software dev has its own group because we've found that topics of that nature tend to usually stay within the boundary, so it's well isolated. I think this pretty much cover your 3rd query.

However, given the recent spike in conversations which are sensitive in nature, might make sense to form a new group after all. I will bring this up with the other Ops member.

2. To improve visibility (at least a little) I've made a tiny change to the lobby group description to include links to other groups. Hopefully that'll be a slight improvement over the usual welcome message.

![Screenshot_20210525-231907_Telegram.jpg](https://user-images.githubusercontent.com/175895/119524890-9c5a7600-bdb0-11eb-9548-d0dc521843fb.jpg)
